src/mediarithmics/api/plugin/audiencefeedconnector/AudienceFeedConnectorPluginResponseInterface.ts

```diff
@@ -38,10 +38,11 @@ type SyncResult =
   | 'PROCESSED'
   | 'FAILURE'
   | 'NO_ELIGIBLE_IDENTIFIER'
-  | 'SUCCESS';
+  | 'SUCCESS'
+  | 'IDENTIFIER_REJECTED';
```

Comment:
REJECTED ?

Reply:
The title of the jira ticket specifier IDENTIFIER_REJECTED.
Do you want to replace it with REJECTED (without IDENTIFIER) or use another word instead of REJECTED?